.gitignore

```diff
@@ -23,8 +22,8 @@ pom.xml.asc
 .lein-deps-sum
 .lein-failures
 .lein-plugins
-.lein-repl-history
-.nrepl-port
```

Comment:
These two lein ones seem as though they should stay in ignore. Why are you removing them?